server/helpers/adapters_tracker.go

```diff
@@ -171,6 +194,17 @@ func (a *AdaptersTracker) UndeployAdapter(ctx context.Context, adapter models.Ad
 			return ErrAdapterAdministration(err)
 		}
 
+	case "kubernetes":
+		kubeClient, err := meshkitkube.New([]byte(""))
+		if err != nil {
+			return ErrAdapterAdministration(err)
```

Comment:
ErrAdapterAdministration() is missing error-specific probable cause and suggested remediation.

So, too, is ErrDockerHost()

Reply:
ErrDockerHost() does have
ErrAdapterAdministration() is generic, similar to ErrNewKubeClient()